data/campaigns/World_Conquest/lua/map/scenario_utils/plot.lua

```diff
@@ -45,29 +45,29 @@ function add_plot(scenario, scenario_num, nplayers)
 	elseif scenario_num == 2 then
 		local r = wesnoth.random(nplayers)
 		start_message(r, true, _ "Ahhh. Just look at these fertile lands, ripe for conquest!")
-		start_message("4", true, _ "What's this?! Foreign invaders have set foot upon our shores!")
-		start_message("5", true, _ "We'll send them back to where they came from quick enough. All troops, to me!")
+		start_message("4", true, _ "What’s this?! Foreign invaders have set foot upon our shores!")
+		start_message("5", true, _ "We’ll send them back to where they came from quick enough. All troops, to me!")
 
 		end_message("1,2,3", false, _ "We win! Truly ours will be a glorious empire!")
 		end_message("1,2,3", true, _ "Do not get hasty! To reach the great continents of the east we must subdue the savage coast of Moragdu, notorious for its corsairs, before we can advance further.")
 	elseif scenario_num == 3 then
-		start_message("4", true, _ "What's this? It is a long time since strangers have set foot upon Moragdu--and in such great numbers too!")
+		start_message("4", true, _ "What’s this? It is a long time since strangers have set foot upon Moragdu--and in such great numbers too!")
```

Comment:
That double-hyphen should change. IIUC, the replacement should be a spaced em dash.

Reply:
Wasn't focused on punctuation as much as spelling, plus everyone seems to follow different conventions. My preference is for spaced en dash while others use non-spaced em dash, but I'll change per your suggestion.